app/views/taxons/show.html.erb

```diff
@@ -25,4 +25,9 @@
     child_taxons: taxon.child_taxons,
     tagged_content: taxon.tagged_content
   } %>
+
+  <%= render partial: 'content_list_for_current_taxon', locals: {
```

Comment:
This seems to be the same in both the `if` and `else` clauses, should we move it outside of the `if` statement?